app/src/main/kotlin/com/x8bit/bitwarden/data/vault/repository/util/VaultSdkCollectionExtensions.kt

```diff
@@ -18,6 +19,8 @@ fun SyncResponseJson.Collection.toEncryptedSdkCollection(): Collection =
         hidePasswords = this.shouldHidePasswords,
         readOnly = this.isReadOnly,
         manage = this.canManage ?: !this.isReadOnly,
+        defaultUserCollectionEmail = null,
+        type = CollectionType.SHARED_COLLECTION,
```

Comment:
Should `this.defaultUserCollectionEmail` and `this.type` be passed through, or are we using hard-coded values for a reason?